lib/src/core/constants/sort_order.dart

```diff
@@ -0,0 +1,8 @@
+enum SortOrder {
+  japanese("japanese"),
+  alphabetical("alphabetical");
+
+  final String value;
+
+  const SortOrder(this.value);
```

Comment:
Avec la dernière version de dart, plus besoin de passer une valeur comme ça juste pour avoir le nom disponible. On peut utiliser le `.name`!

```suggestion
  japanese,
  alphabetical;
```

Pour récuperer le nom en string, il suffit donc maintenant de faire: `SortOrder.japanese.name`

Reply:
Changé